.vscode/settings.json

```diff
@@ -1,13 +1,16 @@
 {
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"[php]": {
-		"editor.defaultFormatter": "esbenp.prettier-vscode"
+		"editor.defaultFormatter": "bmewburn.vscode-intelephense-client"
```

Comment:
Why remove the Prettier plugin?

Reply:
oops I didn't mean to touch this file at all. reverting